src/groovy/org/pih/warehouse/api/StockMovementItem.groovy

```diff
@@ -309,8 +309,7 @@ class AvailableItem {
                 "productCode"     : inventoryItem?.product?.productCode,
                 lotNumber         : inventoryItem?.lotNumber,
                 expirationDate    : inventoryItem?.expirationDate?.format("MM/dd/yyyy"),
-                "binLocation.id"  : binLocation?.id,
-                "binLocation.name": binLocation?.name,
+                binLocation       : binLocation,
```

Comment:
Be careful making changes like this. We probably only use these APIs for the React pages, but we don't for sure. So removing properties like this could break an API client that is expecting these fields to exist. We need to deprecated before we remove.

Reply:
@awalkowiak Thoughts?

Reply:
FWIW I don't want to unnecessarily keep fields around that are deprecated but we should try to formalize a process around removing fields before we do this.

Reply:
@jmiranda it's a good point. I added a cart to the retro board to discuss how would be best to handle such cases.